src/Tizen.TV.UIControls.Forms/Renderer/GridViewRenderer.cs

```diff
@@ -0,0 +1,205 @@
+﻿/*
+ * Copyright (c) 2018 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using ElmSharp;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.ComponentModel;
+using Xamarin.Forms;
+using Xamarin.Forms.Platform.Tizen;
+using Tizen.TV.UIControls.Forms;
+using Tizen.TV.UIControls.Forms.Renderer;
+
+[assembly: ExportRenderer(typeof(GridView), typeof(GridViewRenderer))]
+namespace Tizen.TV.UIControls.Forms.Renderer
+{
+    class GengridItemContext
+    {
+        public object Data { get; set; }
+        public View RealizedView { get; set; }
+    }
+
+    public class GridViewRenderer : ViewRenderer<GridView, GenGrid>
+    {
+        IList<GengridItemContext> itemContexts = new List<GengridItemContext>();
+        GenGrid _genGrid = null;

```

Comment:
No need to use _genGrid variable, because you can use Control
```suggestion
```

Reply:
Fixed it. Thank you.